test/jdk/jdk/jfr/tool/TestMetadata.java

```diff
@@ -44,11 +48,19 @@
 public class TestMetadata {
 
     public static void main(String[] args) throws Throwable {
+        testBasic();
+        testEventTypeNum();
+        testDeterministic();
+        testWildcardAndAcronym();
+    }
+
+    static void testBasic() throws Throwable {
```

Comment:
Perhaps change the name of the method to testUnfiltered() to make it more clear what the test does and create a separate method for the --wrongOption, i.e. testIllegalOption()